staging/src/k8s.io/apiserver/pkg/storage/etcd/metrics/metrics.go

```diff
@@ -59,6 +59,13 @@ var (
 		},
 		[]string{"operation", "type"},
 	)
+	objectCounts = prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Name: "apiserver_object_counts",
```

Comment:
the flag specifies etcd... do we want the metric to be something like `etcd_object_counts` as well?

Reply:
Done.